cni/pkg/install-cni/pkg/install/kubeconfig_test.go

```diff
@@ -0,0 +1,169 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package install
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/coreos/etcd/pkg/fileutil"
+	"github.com/stretchr/testify/assert"
+
+	"istio.io/istio/cni/pkg/install-cni/pkg/config"
+	testutils "istio.io/istio/pilot/test/util"
+)
+
+const (
+	kubeconfigMode = 0600
```

Comment:
[DefaultKubeconfigMode](https://github.com/istio/istio/pull/25332/files?file-filters%5B%5D=.conf&file-filters%5B%5D=.conflist&file-filters%5B%5D=.crt&file-filters%5B%5D=.default&file-filters%5B%5D=.go&file-filters%5B%5D=.golden&file-filters%5B%5D=.mk&file-filters%5B%5D=.sum&file-filters%5B%5D=.template&file-filters%5B%5D=.yaml&file-filters%5B%5D=No+extension#diff-974f3d8dc5b35a4391f4df4a280d13b1R41) from constants can be used here.